seqlike/SeqLike.py

```diff
@@ -31,6 +33,8 @@
     add_seqnums_to_letter_annotations,
     ungap,
 )
+from .Mutation import Mutation, Substitution, Deletion, Insertion
+from .MutationSet import MutationSet, magical_parse
```

Comment:
this can be imported from Mutation directly

Reply:
That doesn't "feel" right, I think -- doing so feels like an indirect import. In this case, an additional line doesn't feel like it would be of too much consequence, and would also have the advantage of being explicit about where the MutationSet class is defined.

Reply:
Sorry, this comment was out of date after you got rid of magical_parse.